lib/importers/google/sessions/saver.rb

```diff
@@ -0,0 +1,29 @@
+module Importers
+  module Google
+    module Sessions
+      class Saver
+        def initialize(records:)
+          @records = records
+        end
+
+        def save
+          ActiveRecord::Base.transaction do
+            appointment_summaries!
+          end
+        end
+
+        private
+
+        def appointment_summaries!
+          @records.each do |record|
+            summary = AppointmentSummary.find_or_initialize_by(record.unique_identifier)
+
+            next if summary.transactions > record.transactions
```

Comment:
This line doesn't seem obvious. At first glance it seems like some business logic that I don't know about.

Perhaps something more like:

```ruby
has_already_been_saved = summary.transactions > record.transactions
if !has_already_been_saved
  summary.update_attributes!(record.params)
end
```
or just:

```ruby
next if has_already_been_saved
```